tests/e2e/tests/devfiles/JavaMaven.spec.ts

```diff
@@ -14,18 +14,19 @@ import * as commonLsTests from '../../testsLibrary/LsTests';
 import * as workspaceHandling from '../../testsLibrary/WorksapceHandlingTests';
 import * as codeExecutionTests from '../../testsLibrary/CodeExecutionTests';
 
-const workspaceName: string = NameGenerator.generate('wksp-test-', 5);
 const sampleName: string = 'console-java-simple';
 const fileFolderPath: string = `${sampleName}/src/main/java/org/eclipse/che/examples`;
 const tabTitle: string = 'HelloWorld.java';
 const codeNavigationClassName: string = 'String.class';
 const stack : string = 'Java Maven';
 const taskName: string = 'maven build';
 
+let workspaceName = 'not defined';
```

Comment:
This variable has no need to be global, assign it in the suite where it's needed

Reply:
The variable is moved to a suite part.